### ISSUE TITLE
Hello, i'm stuck on index pattern process. Please help.

### ISSUE THREAD

**Issue by CAZ1337** [BroadSoft-Xtended/BroadWorks-Dashboards-and-Discovery#34]
I trying to create bwcdr , bwlog to index pattern but end up with "The index pattern you've entered doesn't match any indices." As I understanding I need to put data to elasticsearch for match indices.. I following the document seem like there is no topic about this stuff.

Thank.

**Comment by bchibbard** [comment]
The patterns in Kibana need to be "bwcdr*" and "bwlog*".  In order for Kibana to successfully add the patterns, the indices (for example: bwcdr-201802, bwlog-appserver7-20180201) need to already exist in the elasticsearch cluster that the Kibana server is configured to point to.  If you run
`curl -X GET <youresserverhost-ip>:9200/_cat/indices/bwcdr*?pretty`

you should see something like:

`green open bwcdr-201802 qlnLaKbYQX20hso21bksVP 1 0      0 0    233b    233b`
`green open bwcdr-201801 fQUFW5alTAqcqeezBTiw3Z 1 0 123175 0 155.8mb 155.8mb`

The index patterns need to be created in Kibana **before** you import the dashboard *.json files.

**Comment by CAZ1337** [comment]
OIC, I see the picture now, if I'm not wrong I need to follow on topic 4,5,6 if It was working I will see the indices when I excute the command "curl -X GET <youresserverhost-ip>:9200/_cat/indices/bwcdr*?pretty" 

I will try it, thank you so much.

**Comment by bchibbard** [comment]
That is correct.  Once you have all the data collection in place, you should be in good shape.

You're welcome!

**Comment by CAZ1337** [comment]
seem like when I was trying to run log_template.pl it got error with _all also _search but I changed it to

	"mappings": {
		"log_record": {
			"_source": {
				"enabled": true
and
system("curl -sS -H \"Content-Type: application/json\" --data \@callsequence_template.json -XPUT $hostname:9200/search/template/callsequence_template?pretty");

it's working but not sure it will work or not...

do you have any release form log_template.pl yet?

**Comment by bchibbard** [comment]
Bug fixes for these elasticsearch 6+ template related issues will be released very soon.

**Comment by MistAssassin** [comment]
Any eta on the template fixes?
